nav2_mppi_controller/include/nav2_mppi_controller/models/constraints.hpp

```diff
@@ -41,6 +41,20 @@ struct SamplingStd
   float wz;
 };
 
+/**
+ * @struct mppi::models::AccelConstraints
+ * @brief Constraints on acceleration and deceleration
+ */
+struct AccelConstraints
+{
+  float ax_max;
```

Comment:
Q: Can't these be added to `ControlConstraints` above? I think they count as control constraints

Reply:
Yes, that would be better.